### PR TITLE
Prepare PR for TQ

### PR DESCRIPTION
**Pull request by rmfranken**
Deleted sync-fork action
Fixed \r being an unrecognized character by bash, causing it to not find the entrypoint.sh file. Don't fully understand how this was not a problem before, but it doesn't work without this line in the dockerfile:
```RUN sed -i 's/\r//g' entrypoint.sh```

### CODE REVIEW

File: .docker/Dockerfile [hunk 1]
```diff
@@ -42,5 +42,5 @@ ENV PATH "/app/shacl-${VERSION}/bin:${PATH}"
 COPY --from=jre-build /javaruntime $JAVA_HOME
 COPY --chmod=0755 --from=jre-build /app/shacl-${VERSION} /app/shacl-${VERSION}
 COPY --chmod=0755 --from=jre-build "/app/.docker/entrypoint.sh" "/entrypoint.sh"
-
+RUN sed -i 's/\r//g' entrypoint.sh
```
Comment (cmdoret) [inline .docker/Dockerfile:45]: If possible, I would prefer using a `.gitattributes` file to tell git that this specific file should have a linux file ending (\n).
This being a bash script, it's not meant to be executed in a windows environment anyways.
see e.g.:  https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings?platform=linux#per-repository-settings

**Review by cmdoret — CHANGES_REQUESTED** [review]

File: .docker/Dockerfile (hunk 1 above)
Comment (cmdoret) [reply .docker/Dockerfile:45]: See also: https://stackoverflow.com/questions/71081340/prevent-vscode-to-crlf-files

**Comment by rmfranken** [comment]
Ok, I have it setup so that it makes sure the end of line feeds are always lf if the file is a .sh file. At first I thought to just do it for all files, but maybe that's a little bit over-reachy, as the problem only shows itsself in shellscript for me. What do you think? Do we risk breaking anything by applying it to all files in a repo, or is the "add it to the gitattributes if it causes problems" a better practice? (more fine grained, but also more debugging effort potentially)

**Comment by cmdoret** [comment]
This is looks good, but just to be careful and explicit perhaps we could do:

```
* text=auto
# Shell scripts require LF
*.sh    text eol=lf
# Batch scripts require CRLF
*.bat   text eol=crlf
```
IIUC [the docs](https://git-scm.com/docs/gitattributes#Documentation/gitattributes.txt-Set-1), this means:

* Let git infer which files are text and which are binary. Implicitely, those that are text will always be converted to lf (`\n`) by git.
* Convert `.sh` files to lf (I _think_ this could be omitted)
* Convert `.bat` files to crlf (`\r\n') as they are only executed on windows

**Comment by rmfranken** [comment]
The .bat is a good point! And using auto might actually not require the explicit mentioning of bat since:

_When text is set to "auto", Git decides by itself whether the file is text or binary. If it is text **and the file was not already in Git with CRLF endings**_

The way I interpret that sentence is: If you upload a .bat with CRLF endings, it will remain as such, otherwise, it will be converted to LF. That being said, we can do both - also for documentation sake to make it clear we expect LF endings on non-bat files, and CRLF on bat.

**Review by cmdoret — APPROVED** [review]
Looks good and works on my end :raised_hands:

**Pull request merged**